apps/meteor/app/api/server/v1/cloud.ts

```diff
@@ -122,3 +123,23 @@ API.v1.addRoute(
 		},
 	},
 );
+
+API.v1.addRoute(
+	'cloud.syncWorkspace',
+	{
+		authRequired: true,
+		permissionsRequired: ['manage-cloud'],
+		rateLimiterOptions: { numRequestsAllowed: 2, intervalTimeInMS: 60000 },
+	},
+	{
+		async get() {
+			try {
+				await syncWorkspace();
+
+				return API.v1.success({ success: true });
+			} catch (error) {
+				return API.v1.failure('Error during workspace sync');
+			}
```

Comment:
If I want to send the correct error to the client yes, right? Because the error provided by the upper function is not well written and easy to understand

Reply:
I see the point, but please dont mask the actual error 😬 log it at least

Reply:
It's being logged in the upper func, since it's a old function that has a lot of consumers I'll let it be in this PR

Reply:
Just to be clear, in this case what will be logged in the upper function is not the actual error thrown by the syncWorkspace() fn. 👀